tools/mason/Makefile

```diff
@@ -0,0 +1,40 @@
+
+
+ifndef CHPL_MAKE_HOME
+export CHPL_MAKE_HOME=$(realpath $(shell pwd)/../..)
+endif
+
+include $(CHPL_MAKE_HOME)/make/Makefile.base
+
+bdir=$(CHPL_BIN_DIR)
+link=$(bdir)/mason
+
+
+REGISTRY=https://github.com/Spartee/mason-registry
+
+all: mason install
+
+mason:
+	@mkdir -p ~/.mason
+	@mkdir -p ~/.mason/src
```

Comment:
FYI, ``mkdir -p ~/.mason/src`` will create the ``~/.mason`` directory if it doesn't already exist.